File: Safari-611.1.10/LayoutTests/http/tests/eventsource/resources/reconnect.php
<?php
header("Content-Type: text/event-stream");
$lastEventId = $_SERVER['HTTP_LAST_EVENT_ID'];

if ($lastEventId)
    echo "data: $lastEventId\n\n";
else {
    echo "id: 77\n";
    echo "retry: 300\n";
    echo "data: hello\n\n";
    echo "data: discarded";
}
?>

File: LayoutTests/http/tests/eventsource/resources/reconnect.py
#!/usr/bin/env python3

import os
import sys

sys.stdout.write('Content-Type: text/event-stream\r\n\r\n')

lastEventId = os.environ.get('HTTP_LAST_EVENT_ID')
if lastEventId:
    print('data: {}\n'.format(lastEventId))
else:
    print('''id: 77
retry: 300
data: hello\n
data: discarded''')